src/app/child-dev-project/children/child-details/child-details.component.ts

```diff
@@ -58,13 +57,7 @@ export class ChildDetailsComponent implements OnInit {
     "not eligible",
     "",
   ];
-  eyeStatusValues = ["Good", "Has Glasses", "Needs Glasses", "Needs Checkup"];
-  vaccinationStatusValues = [
-    "Good",
-    "Vaccination Due",
-    "Needs Checking",
-    "No Card/Information",
-  ];
+
```

Comment:
how will we hide fields from the "basic information" section based on config? Do you already have an approach for this?

A temporary solution could be to make that section into a separate component as well and create copies for more than one variant.

Reply:
either way it could be a good idea to move the basic information section into a sub component, that way the responsibility of ChildDetailsComponent is clearer and it is less code in one file.

Reply:
This would probably be the most stable approach. However I would at least try to make an attempt where we define the displayed information in the config file and see how the styling will become. We also should discuss soon, how we define the attributes on the `child` and `school` entity.

Reply:
ok, I think that will happen in a separate, follow-up PR